packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -20,9 +20,12 @@ export const DetailView: FC = () => {
   const { entity, mode, onOpen, onClose, isOpen } = useEditModal<Item>();
   const { data } = useOutbound();
 
-  const onRowClick = (item: InvoiceLine | InvoiceItem) => {
-    onOpen(toItem(item));
-  };
+  const onRowClick = useCallback(
+    (item: InvoiceLine | InvoiceItem) => {
+      onOpen(toItem(item));
+    },
+    [toItem, onOpen]
+  );
```

Comment:
stable reference.. the content area component can re-render and the data table won't so the cost of the re-render in general is cheap